skycultures/tikuna/constellation_names.eng.fab

```diff
@@ -0,0 +1,4 @@
+001	"Baweta" _("Turtle Collective")
+002 "Wücütcha" _("Wücütcha’s")

```

Comment:
A “Jaguar Clan”?

Reply:
that means the jaguar's leg

Reply:
But the line `002 "Wücütcha" _("Wücütcha’s")` should be `002 "Wücütcha" _("Jaguar's leg")` in this case

Reply:
Please check this line

Reply:
This is still an incomplete genitive. @youssifgh ?

Reply:
- Yes it is an incomplete genitive

Reply:
> But the line `002 "Wücütcha" _("Wücütcha’s")` should be `002 "Wücütcha" _("Jaguar's leg")` in this case

- it's not just any jaguar's leg, it's the foot of the mystical jaguar Wücütcha and that's what they call this constellation

Reply:
Should it be "Wücütcha's leg" in this case, or do you really want to keep is as incomplete genitive (which maybe nobody understands)?

Reply:
yes it really makes sense I'm adhering to your suggestion

Reply:
I already made the change, thanks a lot for the suggestion @gzotti